main/template/default/session/about.tpl

```diff
@@ -1,275 +1,292 @@
                                             {% endfor %}
-                                        {% endif %}
-                                    </dl>
-                                </div>
-                            {% endfor %}
-                        </div>
+                                        </p>
+                                    {% endif %}
+                                {% endfor %}
+                            </div>
+                        {% endif %}
                     </div>
                 </div>
             </div>
         </div>
-    {% endif %}
-
-    {% if is_subscribed and user_session_time != -0 and user_session_time >= 1 %}
-        <div class="alert alert-info">
-            {{ 'AlreadyRegisteredToSession'|get_lang }}
-        </div>
-    {% elseif is_subscribed and user_session_time < 1 %}
-        <div class="alert alert-warning">
-            {{ 'YourSessionTimeIsExpired'|get_lang }}
-        </div>
-    {% endif %}
-
+    </div>
     {% for course_data in courses %}
         {% set course_video = '' %}
-
         {% for extra_field in course_data.extra_fields %}
             {% if extra_field.value.getField().getVariable() == 'video_url' %}
                 {% set course_video = extra_field.value.getValue() %}
-            {% endif %}
+        {% endif %}
         {% endfor %}
-        <div class="share-social-media">
-            <ul class="sharing-buttons">
-                <li>
-                    {{ "ShareWithYourFriends"|get_lang }}
-                </li>
-                <li>
-                    <a href="https://www.facebook.com/sharer/sharer.php?{{ {'u': page_url }|url_encode }}"
-                       target="_blank" class="btn btn-facebook btn-inverse btn-xs">
-                        <em class="fa fa-facebook"></em> Facebook
-                    </a>
-                </li>
-                <li>
-                    <a href="https://twitter.com/home?{{ {'status': session.getName() ~ ' ' ~ page_url }|url_encode }}"
-                       target="_blank" class="btn btn-twitter btn-inverse btn-xs">
-                        <em class="fa fa-twitter"></em> Twitter
-                    </a>
-                </li>
-                <li>
-                    <a href="https://www.linkedin.com/shareArticle?{{ {'mini': 'true', 'url': page_url , 'title': session.getName() }|url_encode }}"
-                       target="_blank" class="btn btn-linkedin btn-inverse btn-xs">
-                        <em class="fa fa-linkedin"></em> Linkedin
-                    </a>
-                </li>
-            </ul>
-        </div>
-        <div class="panel-course">
-            <div class="row">
-                {% if courses|length > 1 %}
-                    <div class="col-xs-12">
-                        <h3 class="text-uppercase">{{ course_data.course.getTitle }}</h3>
-                    </div>
-                {% endif %}
+        <div class="panel panel-default panel-course">
```

Comment:
Better to use macros instead of repeat the panels everywhere  ... I think panels changed in bootstrap 4.